release/models/lacp/openconfig-lacp.yang

```diff
@@ -245,6 +251,18 @@ module openconfig-lacp {
         timestamp in nanoseconds relative to the Unix Epoch
         (Jan 1, 1970 00:00:00 UTC).";
     }
+
+    leaf port-priority {
```

Comment:
Per my other comment, I definitely don't think the pattern of having this configuration on `/interfaces` is correct. We should have the `config` and `state` of this leaf mirror each other within the sibling `config` and `state` containers (intended config, applied config in state) rather than having the split across the modules.

Reply:
`/lacp/interfaces/interface` approach should also address this

Reply:
Addressing https://github.com/openconfig/public/pull/943#discussion_r1302002147 will also resolve this conversation